index.ts

```diff
@@ -1,11 +1,17 @@
+import { readFileSync } from 'fs'
+import { jams } from 'jams.js'
 import { PackBuilder } from './src/builder'
 import { Dapp } from './src/dapp'
 import { getIpfsJson, putIpfsJson, isCid } from './src/ipfs-util'
 import { need } from './src/util'
 
 export const load = async (arg, _ethers = undefined, _signer = undefined) => {
-  if (typeof arg === 'string') {
-    arg = (isCid(arg)) ? await getIpfsJson(arg) : require(arg)
+  if (typeof arg === "string") {
```

Comment:
this is a little better but it is still a ternary expression, just write this code like a normal person you don't have to emulate our style if it makes no sense

Reply:
meant line below etc

Reply:
Haha, okay got it.